overlay.d/05rhcos/usr/lib/dracut/modules.d/89ibm2810xiv-mutipath-config/ibm2810xiv-multipath-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Workaround for IBM 2810XIV discs
+
+enable=
+devices=$(lsblk -o name --nodeps --noheadings)
+
+for dev in ${device}; do
+    model=$(cat /sys/class/block/${dev}/device/model)
+    if [ "${model}" == "2810XIV" ]; then
+	enable=1
+	break
+    fi
+done
+
+if [ -n ${enable} ] && [ ! -e /etc/multipath.conf ] ; then
+    mkdir -p /etc/multipath/multipath.conf.d
+    mpathconf --enable
+    sed -i "s/(SCSI_IDENT_|ID_WWN)/(SCSI_IDENT_|ID_WWN|ID_SERIAL)/g" /etc/multipath.conf
```

Comment:
From researching a bit about multipath configs, IIUC what this is doing is augmenting the blacklist exception to include any device which has an `ID_SERIAL` udev property, right? Whereas usually it would only be those with `SCSI_IDENT_...` and `ID_WWN...` properties. Looking at `multipath.conf(5)` from [here](https://github.com/opensvc/multipath-tools/blob/8dcfb54346c110610cccf923000171a10dbf656a/multipath/multipath.conf.5#L1330-L1332), it says:

> The default \fIblacklist exception\fR is: \fB(SCSI_IDENT_|ID_WWN)\fR, causing
well-behaved SCSI devices and devices that provide a WWN (World Wide Number)
to be included, and all others to be excluded.

Is there a concern here that `ID_SERIAL` is too generic and may enable multipath on devices that normally *should* be blacklisted?

Reply:
If this affects a specific model, couldn't we craft a more specific exception instead?

Reply:
So i found the missing part - `scsi-sg3_id.rules`.